tirocks/src/db/imp.rs

```diff
@@ -205,7 +203,9 @@ impl RawDb {
         }
     }
 
-    /// Same as [`get`] but may allocate less.
+    /// Same as [`get`] but store the value in `value`, so it may allocate less.
+    ///
+    /// If such entry is found, value is updated and `Ok(true)` is returned.
     pub fn get_to(
```

Comment:
how about `get_inplace`

Reply:
I rename it to `get_pinned`, so it's aligned with `PinSlice`.